### PR TITLE
Fix unused result

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/discord_register_win.cpp [hunk 1]
```diff
@@ -5,9 +5,9 @@
 #define NOMCX
 #define NOSERVICE
 #define NOIME
-#include <windows.h>
 #include <Psapi.h>
 #include <Strsafe.h>
+#include <windows.h>
```
Comment (crmarsh) [inline src/discord_register_win.cpp]: This part of the change is some clang-format cruft (fixed in another pr) -- as is it would break the windows build.

Comment (janisozaur) [reply src/discord_register_win.cpp]: OK, I will revert this part of the commit, but:

a) Why does header ordering break builds? That sounds like broken headers to me, is it something you can fix?

b) If the first option is not viable, can you add a clang-format guard and explanation to this part?

Comment (janisozaur) [reply src/discord_register_win.cpp]: @crmarsh updated

**Comment by crmarsh** [comment]
Cool, thanks :-)

**Pull request merged**